lib/active_admin/resource/attributes.rb

```diff
@@ -21,7 +21,7 @@ def foreign_methods
         @foreign_methods ||= resource_class.reflect_on_all_associations.
           select { |r| r.macro == :belongs_to }.
           reject { |r| r.chain.length > 2 && !r.options[:polymorphic] }.
-          index_by { |r| r.foreign_key.to_sym }
+          index_by { |r| (r.foreign_key.is_a?(Array) ? r.foreign_key.join('-') : r.foreign_key).to_sym }
```

Comment:
invoking a method here will slow down operations, but the ternary operator hides complexity and codecov is tricked into thinking that this line is being tested.

I would move the conditional to a method and use multiple lines for better code coverage

See warning at https://github.com/activeadmin/activeadmin/pull/8432/files#diff-551e9462ea228e696f89ae7a20c041a4e504d0ab25bad03f2e0c3edae939e344R9

/cc @javierjulio